content/docs/1.4.0/volumes-and-nodes/create-volumes.md

```diff
@@ -48,6 +50,8 @@ When the Pod is deployed, the Kubernetes master will check the PersistentVolumeC
     #  ]'
     ```
 
+    In particular, starting with v1.4.0, the parameter `mkfsParams` can be used to specify filesystem format options for each StorageClass.
```

Comment:
`v1.4.0` hard code, the same.

Reply:
At first I wanted to make it a dynamic value.

But this parameter is added since 1.4.0, if it is a dynamic value, when switching to a future version (e.g., `v1.4.1`), it will be considered that the parameter was added in this version. 

Therefore, I hard code the version number as `v1.4.0` at first.

Reply:
ah, you are right. Sorry for being confused by this. :+1: